main/app.yaml

```diff
@@ -55,9 +55,9 @@ skip_files:
 - ^(.*/)?index\.yaml
 - ^(.*/)?index\.yml
 - ^lib/.*
-- ^static/ext/.*\.coffee
-- ^static/ext/.*\.css
-- ^static/ext/.*\.js
-- ^static/ext/.*\.less
-- ^static/ext/.*\.json
+- ^static/build/ext/.*\.coffee
```

Comment:
As I understand it, we can ignore the this directory completely, like:
```
^static/build/ext/.*
```
Right?